lib/dl_file_uploader_lib/dl_file_uploader_lib/enums.py

```diff
@@ -18,6 +18,7 @@ class FileType(Enum):
     csv = "csv"
     gsheets = "gsheets"
     xlsx = "xlsx"
+    yadocuments = "yadocuments"
```

Comment:
I do not insist, but what do you think about using `ya_docs` for constants/methods and `YaDocs` for classes? This looks a bit too long